src/ServiceControl/Operations/ErrorQueueImport.cs

```diff
@@ -98,7 +98,7 @@ public void Stop()
 
         public Address InputAddress => settings.ErrorQueue;
 
-        public bool Disabled => InputAddress == Address.Undefined;
+        public bool Disabled => InputAddress == Address.Undefined || !settings.IngestErrorMessages;
```

Comment:
*Can* you leave the error address undefined?

Reply:
Yeah. In `Settings.GetErrorQueue` there is conditional.